azext_edge/edge/providers/rpsaas/adr/asset_endpoint_profiles.py

```diff
@@ -95,7 +95,10 @@ def create(
             try:
                 json.loads(configuration)
             except json.JSONDecodeError as e:
-                raise InvalidArgumentValueError(f"Additional configuration is not a valid JSON.\n{e.msg}")
+                raise InvalidArgumentValueError(
+                    "Additional configuration is not a valid JSON. For examples of valid JSON formating, please "
```

Comment:
The way this is implemented right now the inline json example aka.ms link will be shown even if the malformed payload came from a file.